Modules/Tests/FC_Log.Tests.ps1

```diff
@@ -1,4 +1,4 @@
-﻿Remove-Module FC_Log -Force | Out-Null
+﻿Remove-Module FC_Log -Force -ErrorAction SilentlyContinue | Out-Null
```

Comment:
@MarkIannucci, I overhauled the build/test process to use docker, and Updated the Set-LogTargets to throw a non terminating error if you use the windows event log target on pwsh core. 

I added several tests around logging, and I think this behavior is a happy medium for the time being. You get a error/warning that you can ignore if you want when you set-logTargets, and there is some logic in Write-Log so that it will try to use the compatibility mode/actually write to the event log and if it does not it will tell you to checkout the github issue for the workaround